### PR TITLE
Improve implied semantics of setters in GameRules

### PR DESCRIPTION
**Pull request by ChloeDawn**
Also simplifies the public API methods to just get() and set(value)

### CODE REVIEW

**Comment by NeunEinser** [comment]
I wonder why the build fails. Maybe it can't handle multiple `get` methods in the same file? It's different inner classes, so it should definitely be legal.

**Comment by liach** [comment]
>[Caused by: org.apache.http.conn.HttpHostConnectException: Connect to libraries.minecraft.net:443 [libraries.minecraft.net/13.249.133.78] failed: Connection timed out (Connection timed out)](https://travis-ci.com/FabricMC/yarn/builds/114879684#L614)

@InsomniaKitten Consider rebuilding on the travis ci page? As a pr author, you can rebuild your pr on travis ci. https://travis-ci.com/FabricMC/yarn/builds/114879684

**Review by NeunEinser — APPROVED** [review]

**Comment by ChloeDawn** [comment]
I do not see a way to rebuild, even when signed in... Oh well, the build failure is unrelated to the changes made here.

File: mappings/net/minecraft/world/GameRules.mapping [hunk 1]
```diff
@@ -62,15 +62,15 @@ CLASS bhh net/minecraft/world/GameRules
 		FIELD a type Lbhh$e;
 		METHOD <init> (Lbhh$e;)V
 			ARG 1 type
-		METHOD a readValue (Lcom/mojang/brigadier/context/CommandContext;Ljava/lang/String;)V
+		METHOD a setFromArgument (Lcom/mojang/brigadier/context/CommandContext;Ljava/lang/String;)V
 			ARG 1 context
 			ARG 2 name
```
Comment (liach) [inline mappings/net/minecraft/world/GameRules.mapping:67]: Is this the name for the argument

**Review by liach — APPROVED** [review]

**Pull request merged**